### DISCUSSION TITLE
Arguments for and against Earth as a special place for harboring life

### DISCUSSION THREAD

**Discussion by LifeIsStrange** [a-cosmology-group/acg#251] [Astrophysics and Physics]
https://arxiv.org/abs/astro-ph/0112399

**Comment by LifeIsStrange** [comment]
Possible non abiotic mechanism for oxygen generation in oceans
https://news.ycombinator.com/item?id=41036526

**Reply by ExpEarth** [reply]
A news story about it here.  They call it 'dark oxygen'.  This is a dramatic development for astrobiology and origins of life.
(https://phys.org/news/2024-07-metallic-minerals-deep-ocean-floor.html?utm_source=nwletter&utm_medium=email&utm_campaign=daily-nwletter)

**Reply by LifeIsStrange** [reply]
This is very interesting as oxygen is the main (but not only) source of energy generation, however I do not believe (could be wrong) life would have started as aerobic, because while improved energy generation catalyze the possibility of life to appear and maintain itself, aerobic phosphorylation is highly oxidative, having studied gerontology, I know our cells do not suffer from sudden death only because of a complex arsenal of antioxidative defenses.
Though maybe life appeared in a naturally highly reductive milieu (filled with non abiotic antioxidants), what could such a milieu be? 
I've heard free hydrogen is antioxidative but not sure if it is signaletically or chemically.
here is an interesting theory https://www.liebertpub.com/doi/full/10.1089/ast.2019.2156

**Reply by ExpEarth** [reply]
You have pretty good instincts on this.  I agree with you that the origins of life most likely occurred under anaerobic conditions.  I think that hydrogen was most likely the reducing agent.  It seems that hydrogen can also be generated as 'geological hydrogen' from certain mantle rocks (e.g., https://www.sciencedirect.com/science/article/pii/0012821X83901449 ).  The origins process likely took place on mineral surfaces, most likely metal sulfides (eg pyrite).  Pyrite is a known phototransducer.  Another reason life most likely evolved on mineral surfaces is that protein catalysts were not initially present.  If the first biomolecules arose at certain positions related to catalytic sites on mineral surfaces, however, then reaction sequences could have occurred even without specific enzymes.

On the oxygen question, I was never too convinced that microbial photosynthesis could have created the oxygen levels we have now.  However, if there was a preexisting abiotic process, like the one you reported, then early microbes might simply have exploited that on the way to photosynthesis.

**Reply by LifeIsStrange** [reply]
> It seems that hydrogen can also be generated as 'geological hydrogen' 


Interesting, I'd also like to add that giant gas planets is a selection bias from our solar system, I don't remember the exact statistics but rocky vs gasous hydrogen super-earths we observe might be as frequent. It might be that ~half of earth like planets are filled with hydrogen.
Moreover since the moon most likely formed via a giant collision, it might be that such an event made a previous atmosphere hydrogen layer "exit" earth.

A potentially interesting effect in minerals would be https://en.wikipedia.org/wiki/Piezoelectricity

**Reply by ExpEarth** [reply]
@LifeIsStrange 

Concerning the Moon, it most surely did _not_ arise in a collision event.  The hypothesis of a giant impactor (Theia) which hit the early Earth and formed the Moon was cooked up by a handful of geologists decades ago.  It never made much sense.  This is another case of scientists having a sheep mentality.  Geologists can behave very much like cosmologists at times, but because they are more grounded in reality they are not as bad.

The isotopic composition of rocks from the Moon's surface is almost exactly the same as the Earth's mantle.  There is _no_ evidence of the Theia impactor in the rocks.  The Moon must have arisen by ejection of material from the Earth's mantle.  The classic hypothesis is Darwin's fission hypothesis.  If the early Earth rotated fast enough, a piece of it would have been shot out into orbit.  The problem with Darwin's hypothesis is that the angular momentum of the early Earth would have to have been 4 times greater than the total angular momentum in the Earth-Moon system today.  This led to scientists dropping fission.  However, the hypothesis can be saved if there is an explosive release of energy that assists the fission process.  Then the Earth's early angular momentum could have been the same as today's (Earth + Moon).  Attached is a paper by de Meijer et al that discusses all this and proposes such a mechanism.  They suppose a uranium georeactor at the core-mantle boundary was the energy source.  This mechanism is unrealistic, however.  The real mechanism likely involves the Hubble luminosity.
[Moon Origin_de Meijer_Forming-the-Moon-Chemical-Geolog.pdf](https://github.com/user-attachments/files/16393670/Moon.Origin_de.Meijer_Forming-the-Moon-Chemical-Geolog.pdf)
